models/src/main/protobuf/ProposeServiceV1.proto

```diff
@@ -20,6 +20,7 @@ option (scalapb.options) = {
 
 service ProposeService {
   rpc propose(PrintUnmatchedSendsQuery) returns (ProposeResponse) {}
+  rpc proposeResult(ProposeResultQuery) returns (ProposeResultResponse) {}
```

Comment:
Currently we might need to change the propose script for this.

To be very honest, I don't want this feature in this version right now. Propose script is very stable right now after several modifications.

we can discuss this after this pr.

Reply:
Also empty message is already defined in protobuf.
https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/empty.proto